jekyll/_cci2_ja/aws.adoc

```diff
@@ -265,10 +265,10 @@ NOTE: 管理コンソールで、 `Failure reported from operator: no such image
 //<!--add info on making users administrators etc. to user management section of ops guide and put a link here-->
 +
 .ダッシュボードから CircleCI を起動する
-image::dashboard.png[]
+image::dashboard.png[CircleCI server dashboard]
 
 . ビルドコンテナが起動してイメージがダウンロードされると、すぐに最初のビルドを開始します。 **約 15 分**が経過し、[Refresh] ボタンをクリックしても更新が行われない場合は、 https://support.circleci.com/hc/ja[CircleCI サポート]にお問い合わせください。
 
 . 次に、 https://github.com/circleci/realitycheck[realitycheck レポジトリ]を使用して、基本的な CircleCI 機能を確認します。
 
-. 最初のビルドの実行に失敗する場合は、まず https://circleci.com/docs/2.0/troubleshooting[トラブルシューティング]ガイドで一般的なトラブルシューティングのトピックを参照してください。CircleCI Server 内の Builder の状態を確認する方法については、 https://circleci.com/docs/2.0/nomad[Nomad クラスタの操作ガイド]を参照してください。
\ No newline at end of file
+. 最初のビルドの実行に失敗する場合は、まず https://circleci.com/docs/2.0/troubleshooting[トラブルシューティング]ガイドで一般的なトラブルシューティングのトピックを参照してください。CircleCI Server 内の Builder の状態を確認する方法については、 https://circleci.com/docs/2.0/nomad[Nomad クラスタの操作ガイド]を参照してください。
```

Comment:
Adding `/ja` for link might be nice for customer.

- https://circleci.com/docs/ja/2.0/troubleshooting
- https://circleci.com/docs/ja/2.0/nomad